components/pipeline-service/development/main-pipeline-service-configuration.yaml

```diff
@@ -2029,7 +2029,7 @@ spec:
                 key: endpoint
         tolerations:
           - effect: NoSchedule
-            key: node-role.kubernetes.io/master
+            key: konflux-ci.dev/workload
```

Comment:
The pipelines will only run on the nodes with label "konflux-ci.dev/workload: konflux-tenants". The current change will setup log forwarder on both the nodes where we run konflux services "konflux-ci.dev/workload: konflux-services"  and where we run pipelines "konflux-ci.dev/workload: konflux-tenants".

I am just letting you know in case you want to deploy the log forwarder only on the nodes where we run tekton pipelines, then you will need to update this toleration to check value of label

Reply:
Yes, we are interested only in `konflux-tenants`.

Reply:
I have updated with that value.